src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -9,7 +9,7 @@
 
 import numpy as np
 
-from .slope_fitter import ols_slope_fitter  # c extension
+#from .slope_fitter import ols_slope_fitter  # c extension
```

Comment:
Why is this commented out?

Reply:
This causes an unresolved reference for both .slope_fitts and ols_slope_fitter. I've removed the comment.